lint-requirements.txt

```
@@ -0,0 +1,4 @@
+black>=24.4.2
+clang-format>=18.1.5
+ruff>=0.4.4
+yamllint>=1.35.1
```

```suggestion
clang-format>=18.1.5
yamllint>=1.35.1
```

We don't need `black` nor `ruff` since we don't have any Python code in this repo.